src/style.css

```
@@ -1 +1,62 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", serif;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+body {
+  margin: 0;
+  background-color: #eee;
+}
+
+header {
```

add a class to the header and then use it.
```suggestion
.header {
```